predicates:
  at:
    max_distance: 1.0
  on-pred:
    max_above: 0.05
explorer:
  max_sample_repetitions: 1
  max_sequence_length: 2
  max_samples_per_sequence_length: 100
  max_failed_samples: 50
  bounding_box_inflation_length: 0.2
  action_denylist: []
  predicate_precondition_allowlist: ["inside", "on"]
  radii: [0.1, 1.5, 10.0]
robot_arm:
  max_force_magnitude: 150
grasping:
  pregrasp_z_offset: 0.15
